.github/workflows/tests.yml

```diff
@@ -15,6 +15,7 @@ jobs:
     runs-on: ${{ matrix.os }}
     env:
       GOPATH: ${{ github.workspace }}
+      BASE_PATH: ${{ github.workspace }}/src/github.com
```

Comment:
Project support now go modules, so this env will allow local setup to customize the source path.